src/pages/help-centre/deposits-and-withdrawals.js

```diff
@@ -109,6 +76,7 @@ const LiftWithdrawalLimits = () => (
                 components={[
                     <strong key={0} />,
                     <ExternalLink
+                        to=""
```

Comment:
```suggestion
                        to={`${deriv_app_url}/account/account-limits`}
```
instead of using the href?

Reply:
Nice catch 🥳